sql/revenue_derived/client_ltv_v1/query.sql

```diff
@@ -215,14 +215,28 @@ with_caps AS (
 with_ltv AS (
   SELECT
     *,
-    ad_click_days * ad_clicks_per_day_capped * avg_client_ad_click_value AS ltv_ad_clicks_current,
-    search_with_ads_days * searches_with_ads_per_day_capped * avg_client_search_with_ads_value AS ltv_search_with_ads_current,
-    search_days * searches_per_day_capped * avg_client_search_value AS ltv_search_current,
-    tagged_search_days * tagged_searches_per_day_capped * avg_client_tagged_search_value AS ltv_tagged_search_current,
-    pred_num_days_clicking_ads * ad_clicks_per_day * avg_client_ad_click_value AS ltv_ad_clicks_future,
-    pred_num_days_seeing_ads * searches_with_ads_per_day_capped * avg_client_search_with_ads_value AS ltv_search_with_ads_future,
-    pred_num_days_searching * searches_per_day_capped * avg_client_search_value AS ltv_search_future,
-    pred_num_days_tagged_searching * tagged_searches_per_day_capped * avg_client_tagged_search_value AS ltv_tagged_search_future,
+    SQRT(
```

Comment:
@benmiroglio did you mean to use the sqrt of the capped values?

Reply:
Yes, the capped values are what we want here, just an added layer of outlier-reduction.